app/resources/api/v1/development_resource.rb

```diff
@@ -17,6 +17,8 @@ class DevelopmentResource < JSONAPI::Resource
       boolean_filters :rdv, :asofright, :ovr55, :clusteros, :phased,
         :stalled, :cancelled, :hidden
 
+      filter :status
```

Comment:
Sounds good. I'm removing these attributes from the controller until then. Thanks:

```
detail: "name is not allowed.",
detail: "address is not allowed.",
detail: "municipality is not allowed.",
detail: "saved is not allowed."
```